quickstart.md

```diff
@@ -191,7 +211,7 @@ documentation:
 - [Virtual workspaces](https://github.com/kcp-dev/kcp/tree/main/docs/content/en/main/concepts/virtual-workspaces.md) - details on kcp's mechanism for virtual views of workspace content
 
 ### KCP-Edge
-TBD
+- [Edge scheduler](https://github.com/kcp-dev/edge-mc/blob/main/docs/poc2023q1/edge-scheduler.md) - demo of edge scheduler functionality
```

Comment:
Maybe better to put this link in some additional section (say "Try KCP-Edge Components") after "Build and Install KCP-Edge"? These components, including the edge scheduler, are in early stage, so their launching are not automated (yet).

@clubanderson I think we can put a link to https://docs.kcp-edge.io/ at this location instead. What do you think?

Reply:
the quickstart.md is linked to by the kcp-edge.io in the https://docs.kcp-edge.io/docs/overview/.  I need to look into how to add a native file that is not in the docs path.  For now, make the necessary adjustments to the root quickstart.md.